src/math/max.js

```diff
@@ -0,0 +1,18 @@
+const max = (numArray) => {
+let currentHighNumber = 0;
+  if(numArray[0] > numArray[1]) {
+    currentHighNumber =numArray[0];
+  } else if (numArray[1] > numArray[0]) {
+    currentHighNumber = numArray[1];
+    }
```

Comment:
Lines 1 through 7 are completely redundant. You're checking the two first values in the array manually then using a for loop starting at index 2 later on.

Just start your for loop at index 0.

```
for(let i = 0; i < numArray.length; i++) {
  // do the things 
}
```